pennylane/qnodes/base.py

```diff
@@ -740,11 +721,13 @@ def evaluate(self, args, kwargs):
 
         self.device.reset()
 
+        temp = self.properties.get('use_native_type', False)
         if isinstance(self.device, qml.QubitDevice):
-            ret = self.device.execute(self.circuit)
+            ret = self.device.execute(self.circuit, return_native_type=temp)
```

Comment:
Nice, so this is all that was necessary to remove the need for `PassThruQNode` to reimplement `evaluate()`?

Reply:
That, and giving PassthruQNodes an `output_conversion` that does nothing, `lambda x: x`. But I had to make "use_native_type" an extra property for all QNode classes, so it's a tradeoff.